poster/src/post_with_retries.ts

```diff
@@ -55,6 +59,14 @@ async function postWithRetries(transaction: TransactionConfig, signerKey: string
       };
     }
 
+    if (maybeIsOutOfGas(e)) {
```

Comment:
I added a gas increase ability for the situation when a gas estimate is not very accurate